charts/replicated-library/values.yaml

```diff
@@ -33,3 +33,9 @@ defaults:
       successThreshold: 1
       failureThreshold: 5
     startupProbe: {}
+
+troubleshoot:
+  replicated:
+    type: SupportBundle
+    enabled: true
```

Comment:
I like where you've defined the defaults but let's set enabled to `false` in the chart so the user can turn it on.

Reply:
changed